packages/backend/src/controllers/authentication.ts

```diff
@@ -64,6 +64,20 @@ export const getOpenIdUserFromRequest = ({
     return undefined;
 };
 
+export const generateOktaUrl = (path: string): string => {
```

Comment:
Could we replace this string manipulation with `path.posix`:

```typescript
const fullPath = path.posix.join(
  path.posix.sep,
  'oauth2',
  lightdashConfig.auth.okta.authorizationServerId || '', // empty string will be skipped
  'v1',
  path,
)

return new URL(fullPath, `https://${lightdashConfig.auth.okta.oktaDomain}`)
```

Reply:
Yep, that's much cleaner 😄